index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Easy UI</title>
+    <link rel="stylesheet" href="./index.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/v4-shims.css">
+    
+</head>
+<body>
+    <div class="index">
+        <header class="header">
+            <h1>Easy<span class="logo">UI</span></h1>
+        </header>
+        <main class="main">
+            <br><br><br>
+            <h1>Simple, Easy, Customizable and Open Source Component Library</h1>
+            <br><br><br>
+            <button class="btn btn-primary"><a href="./docs.html">Get Started</a></button>
+            <br><br><br>
+            <h2>copy & paste the below link to your CSS file</h2>
+            <br><br><br>
+            <div class="link">
+                <h3><link rel="stylesheet" href="https://easy-ui7.netlify.app/styles.css">&ltlink rel="stylesheet" href="https://easy-ui7.netlify.app/styles.css"></h3>
+            </div>
+            <br><br><br>
```

Comment:
Instead of writing br for spacing, I would suggest to use css margin property and specify the spacing.